docs/out/actionability.md

```diff
@@ -11,13 +11,13 @@ Some actions like `page.click()` support `{force: true}` option that disable non
 
 | Actions | Performed checks |
 | ------ | ------- |
-| `check()`<br>`click()`<br>`dblclick()`<br>`tap()`<br>`uncheck()` | [Visible]<br>[Stable]<br>[Enabled]<br>[Receiving Events]<br>[Attached] |
-| `hover()` | [Visible]<br>[Stable]<br>[Receiving Events]<br>[Attached] |
-| `fill()` | [Visible]<br>[Enabled]<br>[Editable]<br>[Attached] |
-| `dispatchEvent()`<br>`focus()`<br>`press()`<br>`setInputFiles()`<br>`selectOption()`<br>`type()` | [Attached] |
-| `scrollIntoViewIfNeeded()`<br>`screenshot()` | [Visible]<br>[Stable]<br>[Attached] |
-| `selectText()` | [Visible]<br>[Attached] |
-| `getAttribute()`<br>`innerText()`<br>`innerHTML()`<br>`textContent()` | [Attached] |
+| `check()`<br></br>`click()`<br></br>`dblclick()`<br></br>`tap()`<br></br>`uncheck()` | [Visible]<br></br>[Stable]<br></br>[Enabled]<br></br>[Receiving Events]<br></br>[Attached] |
```

Comment:
nit: `<br>` is a self-closing tag, so we might as well not add `</br>`.

Reply:
mdx requires it to be xml, not html...